packages/story-editor/src/components/keyboardShortcutsMenu/index.js

```diff
@@ -46,12 +46,7 @@ const StyledToggleButton = styled(ToggleButton)`
 `;
 
 const Wrapper = styled.div`
-  /**
-    * sibling inherits parent z-index of Z_INDEX.EDIT
-    * so this needs to be placed above that while still
-    * retaining its position in the DOM for focus purposes
-    */
```

Comment:
🤔 Should this comment move with the value too?

Reply:
yes